### PR TITLE
Cleanup Azure Pipelines configuration

### PR DESCRIPTION
**Pull request by chrisrpatterson**
- adjusted config to use latest syntax
- cleaned up formatting in general

### CODE REVIEW

File: vsts.yml [hunk 1]
```diff
@@ -1,78 +1,71 @@
...
-          yarn install --force
-        name: Install
-      - script: |
-          yarn build:prod
-        name: Build
-      - script: |
-          yarn test:setup && yarn test
-        name: Test
-
-  - phase: Lint
```
Comment (shiftkey) [inline vsts.yml]: Looks like our new lint task from #5602 was lost in that last merge (which means we don't lint on the other agents)

Comment (chrisrpatterson) [reply vsts.yml]: Yep I am fixing that

**Comment by shiftkey** [comment]
@chrisrpatterson I just threw a lint commit on top of the branch because our prettier config also runs on YAML files, which was clashing with your changes

**Comment by chrisrpatterson** [comment]
@shiftkey can you go in and rerun the azure-pipelines ci?  The ubuntu image issue should be resolved.

**Comment by shiftkey** [comment]
@chrisrpatterson done

**Comment by shiftkey** [comment]
@chrisrpatterson the Linux agents still seem to be failing - is there a different agent that will work, or should we hold off until this is resolved?

**Comment by chrisrpatterson** [comment]
@shiftkey I will have one of the engineers take a look, this issues should have been resolved by now.

File: vsts.yml [hunk 2]
```diff
@@ -18,8 +19,9 @@ phases:
...
         name: Test
 
-  - phase: Linux
-    queue: Hosted Linux Preview
+  - job: Linux
+    pool:
+      vmImage: ubuntu-16.04
     steps:
       - script: |
           apt-get update
```
Comment (chrisrpatterson) [inline vsts.yml]: Ok this is  my bad this should be 

sudo apt-get update

File: vsts.yml [hunk 3]
```diff
@@ -58,8 +61,9 @@ phases:
...
         name: Test
 
-  - phase: Lint
-    queue: Hosted Linux Preview
+  - job: Lint
+    pool:
+      vmImage: ubuntu-16.04
     steps:
       - script: |
           apt-get update
```
Comment (chrisrpatterson) [inline vsts.yml]: Should be sudo apt-get update

**Comment by shiftkey** [comment]
Ready for review :apple:

**Pull request merged**